src/feature/client/transports.c

```diff
@@ -1439,6 +1439,37 @@ create_managed_proxy_environment(const managed_proxy_t *mp)
    */
```

Comment:
If you'd like to remove the practracker exception for this function entirely, you could split out the address/port/proxy settings into another function. And leave the transport/state/other settings.

Just an idea. Not sure how serious we are about practracker.